### DISCUSSION TITLE
how to use ungoogled-chromium on a chromium-using-browser

### DISCUSSION THREAD

**Discussion by ThDag** [Eloston/ungoogled-chromium#2465] [Q&A] [answered]
i like ungoogled chromium but i want to use it for a browser that uses chromium. can i just switch the chromium for ungoogled-chromium?

is there any way to achive that or not?

thats for any response

**Comment by PF4Public** [comment]
See https://github.com/ungoogled-software/ungoogled-chromium/discussions/2402

**Reply by ThDag** [reply]
thanks for the response but thats not what i mean 
i mean; how can i change chromium for ungoogledchromium in any of these browsers https://en.m.wikipedia.org/wiki/Chromium_(web_browser)#Browsers_based_on_Chromium

**Reply by networkException** [reply, marked as answer]
You cannot do that with closed source browser and it's very unlikely that it would work with any open source browser either. ungoogled-chromium is not meant to be compatible with non vanilla chromium.
